db/patch68.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE `talk_types_link` (
```

Comment:
Not sure if this is the correct way to do DB changes ? Its basically these two tables